pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>ir.sahab</groupId>
     <artifactId>hbase-rule</artifactId>
-    <version>1.0.1</version>
+    <version>1.0.2</version>
```

Comment:
According to semantic versioning rules your new version is 1.1.0

See https://semver.org/ if you want to check it